app/package.json

```diff
@@ -2,7 +2,7 @@
   "private": true,
   "scripts": {
     "dev": "NODE_OPTIONS='--inspect' next dev",
-    "build": "next build",
+    "build:next": "next build",
```

Comment:
Was there a reason the command needed to be changed from `build` to `build:next`?

Reply:
that is what the original ciip dockerfile ran, if it make no difference I'll change it back to build